app/workers/reindex_alegre_workspace.rb

```diff
@@ -68,7 +71,9 @@ def check_for_write(running_bucket, event_id, team_id, write_remains=false, in_p
     # manage dispatch of documents to bulk similarity api call in parallel
     if running_bucket.length > 500 || write_remains
       log(event_id, 'Writing to Alegre...')
-      Parallel.map(running_bucket.each_slice(30).to_a, in_processes: in_processes) { |bucket_slice| Bot::Alegre.request('post', '/text/bulk_similarity/', { documents: bucket_slice }) }
+      running_bucket.each do |item|
+        Bot::Alegre.query_async_with_params(item[:doc], item[:type])
+      end
```

Comment:
Thanks for the description in the PR - I see we're moving from bulk to single here... worth leaving a `# FIXME` in the code.

Reply:
yep agreed - will toss that in then merge @caiosba